src/ostorlab/cli/vulnz/list/list.py

```diff
@@ -13,10 +14,37 @@
 
 @vulnz.command(name="list")
 @click.option("--scan-id", "-s", "scan_id", help="Id of the scan.", required=True)
+@click.option(
+    "--risk-rating", "-r", "risk_rating", help="Filter vulnerabilities by risk rating."
+)
+@click.option(
+    "--filter-type",
+    "-f",
+    "filter_type",
+    type=click.Choice(["exact", "gte", "lte"]),
+    help="Specify the filter type for risk rating (exact, gte for greater than or equal, lte for less than or equal).",
+)
+@click.option("--title", "-t", help="Filter vulnerabilities by title.")
```

Comment:
Also, remove `--title` and use simple `search` where we do simple careless search on all content.